BESMod/Systems/Hydraulical/Transfer/RadiatorPressureBased.mo

```diff
@@ -10,7 +10,7 @@ model RadiatorPressureBased "Pressure Based transfer system"
     Q_flow_design={if use_oldRad_design[i] then QOld_flow_design[i] else Q_flow_nominal[i] for i in 1:nParallelDem},
     TTra_design={if use_oldRad_design[i] then TTraOld_design[i] else TTra_nominal[i] for i in 1:nParallelDem});
 
-  parameter Boolean use_oldRad_design[nParallelDem]=fill(false, nParallelDem)
+  parameter Boolean use_oldRad_design[nParallelDem]={if QOld_flow_design[i]==Q_flow_nominal[i] then false else true for i in 1:nParallelDem}
     "If true, radiator design of the building with no retrofit (old state) is used"
```

Comment:
Is this necessary? If they are the same, Q_flow_nominal will be used. If not, false leads to Q_flow_nominal being used anyway. So in both cases _nominal is used?